components/status/StatusBar.vue

```diff
@@ -223,6 +267,28 @@ export default {
         Object.keys(this.protocol_assignments).includes(well.toString())
       );
     },
+    is_playback_active: function () {
+      return [STATUS.MESSAGE.LIVE_VIEW_ACTIVE, STATUS.MESSAGE.RECORDING, STATUS.MESSAGE.BUFFERING].includes(
+        this.status_uuid
+      );
+    },
+    is_initializing: function () {
+      return [
+        STATUS.MESSAGE.SERVER_READY,
+        STATUS.MESSAGE.SERVER_STILL_INITIALIZING,
+        STATUS.MESSAGE.INITIALIZING_INSTRUMENT,
+        STATUS.MESSAGE.CALIBRATING,
+        STATUS.MESSAGE.CHECKING_FOR_UPDATES,
+        STATUS.MESSAGE.INSTALLING_UPDATES,
```

Comment:
Might want to separate out everything update related except for "CHECKING_FOR_UPDATES" since it's not really initializing at that point, but it's fine for now

Reply:
Once it enters the update flow it has to be restarted before you can calibrate or do anything else